api/turing/service/router_version_service.go

```diff
@@ -31,6 +37,9 @@ type RouterVersionsService interface {
 	FindLatestVersionByRouterID(routerID models.ID) (*models.RouterVersion, error)
 	// Delete Deletes the given RouterVersion from the db. This method deletes all child objects (enricher, ensembler).
 	Delete(routerVersion *models.RouterVersion) error
+	// FindRouterVersionsByEnsembler Finds router version with status parameter
+	// that use ensembler with id = ensemblerID
+	FindRouterVersionsByEnsembler(options RouterVersionByEnsemblerListOptions) ([]*models.RouterVersion, error)
```

Comment:
Same comment here about changing the service method name to make it more consistent with that of the new endpoint name (i.e. we don't need to specify the 'filtering by ensembler' functionality in the service method name).